source/browse_repository.rst

```diff
@@ -35,26 +54,96 @@ commit with matching text. You can search for the next or previous commit with m
 
 In ``Settings``, ``Git Extensions`` you can change the timeout for typing the text for the quick search.
 
+Navigation
+^^^^^^^^^^
 Go to a specific commit
-^^^^^^^^^^^^^^^^^^^^^^^
+"""""""""""""""""""""""
 
 You can jump to a particular commit in the commit history if you know the SHA, tag or branch. In fact you can use any
 expression valid for git-rev-parse. Select ``Navigate``, ``Go to commit`` or press ``Ctrl-Shift-G`` to open the ``Go
 to commit`` window. Enter an SHA or other term to be passed to git-rev-parse into the box at the top and click ``Go``,
 or select a branch or tag from one of the two combo boxes below.
 
+Filter
+^^^^^^
+
+You can also filter the commit history so that fewer commits are shown.
+
 Filter history
-^^^^^^^^^^^^^^
+""""""""""""""
 
 The history can be filtered using regular expressions and basic filter terms. Filtering will reduce the number of commits
 that are shown in the Git Extensions commit history. The quick filter in the toolbar filters by the commit message, the
 author and/or the committer.
 
 .. image:: /images/quick_filter.png
 
+.. _filter-file:
+
 In the context menu of the commit log you can open the advanced filter dialog. The advanced filter dialog allows you to
 filter for more specific commits. To remove the filter either remove the filter in the toolbar and press enter or remove the
 filter in the advanced filter dialog.
 
 .. image:: /images/advance_filter_dialog-menu.png
+
+When a filter is set, the icon for the advanced filter is changed and you see the current filter as a tooltip for the button.
+
 .. image:: /images/advance_filter_dialog.png
+
+.. _browse-reflog:
+
+Reflog
+""""""
+
+By default Git will not show commits that has no branch or tag on a child. Such commits will be purged when Git runs maintenance.
+By enabling Reflog (toolbar button or ``View`` menu) these commits will be shown too.
+
+Tabs
+----
+
+:ref:`settings-tabs`
+
+Commit
+^^^^^^
+
+Shows conformation or the currently selected commit.
+This tab can be moved to the revision grid.
```

Comment:
Typos? I'm struggling with this...

Reply:
`Commit information and message as well as branch and tag information for the selected commit.`

Especially this page in the doc should be improved. This PR is hopefully enough improvements to not be embarrassing.
I may make another stab at it some time, but no short term plans.